Cargo.toml

```diff
@@ -8,4 +8,9 @@ members = [
 ]
 
 [patch.crates-io]
+h2 = { path = "../h2" }
 prost-derive = { git = "https://github.com/danburkert/prost" }
+
+[patch."https://github.com/tower-rs/tower-h2"]
+tower-h2 = { path = "../tower-h2" }
```

Comment:
are these patches still necessary? this seems like a blocker

Reply:
Welp, just came back and realized I added a comment about these, but never clicked "submit review". Here it is:

> These were here so that we could confirm the goaway stuff does indeed work in h2 before pushing a release. We'll need a new release and then these can be removed (and the lockfile fixed).